projects/components/src/skeleton/skeleton.component.ts

```diff
@@ -17,6 +19,10 @@ import { ChangeDetectionStrategy, Component, Input, OnChanges } from '@angular/c
         </div>
         <div *ngSwitchDefault [ngClass]="this.containerClass"></div>
       </ng-container>
+
+      <ng-template #imageSkeletonTemplate>
+        <img [src]="this.imagePath" [ngClass]="this.skeletonType" />
```

Comment:
once you add the svgs to IconModule, we should be able to use ht-icon for this. We may have to choose a custom size -> set it to 100% 100%. Yes, the name of the component sounds weird for this as the loader would take up the full page and not an icon.